{
  "uid" : "c7efec0d4fcde38b",
  "name" : "Verify redirection to used resources page not authorized user.",
  "fullName" : "tests.main_page_test.TestMainPage#test_mp_08_verify_redirection_to_used_resources_page",
  "historyId" : "43df35c0ef3ab6719310f8b9b9ee7530",
  "time" : {
    "start" : 1738029413444,
    "stop" : 1738029453868,
    "duration" : 40424
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
  "statusTrace" : "self = <pages.main_page.MainPage object at 0x7f8f5410de20>\nurl = 'https://brainup.site/used-resources'\n\n    def check_expected_link(self, url):\n        with allure.step(f'Check url is present: {url}'):\n            try:\n>               return Wait(self.driver, self.timeout).until(\n                    ec.url_to_be(url), message=f\"Can't find element by locator {url}\")\n\npages/base_page.py:68: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"92f1b8fbdabceb08c257bb1f8f32bdca\")>\nmethod = <function url_to_be.<locals>._predicate at 0x7f8f540ff240>\nmessage = \"Can't find element by locator https://brainup.site/used-resources\"\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources\n\n/opt/hostedtoolcache/Python/3.12.8/x64/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException\n\nDuring handling of the above exception, another exception occurred:\n\nself = <tests.main_page_test.TestMainPage object at 0x7f8f542f1310>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"92f1b8fbdabceb08c257bb1f8f32bdca\")>\nmain_page_open = None\n\n    @allure.title(\"Verify redirection to used resources page not authorized user.\")\n    def test_mp_08_verify_redirection_to_used_resources_page(self, driver, main_page_open):\n        page = MainPage(driver)\n>       resources_page_url = page.open_used_resources_page()\n\ntests/main_page_test.py:57: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/main_page.py:68: in open_used_resources_page\n    self.check_expected_link(MainPageLinks.URL_USED_RESOURCES_PAGE)\npages/base_page.py:72: in check_expected_link\n    return Wait(self.driver, self.timeout).until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"92f1b8fbdabceb08c257bb1f8f32bdca\")>\nmethod = <function url_to_be.<locals>._predicate at 0x7f8f540ff060>\nmessage = \"Can't find element by locator https://brainup.site/used-resources\"\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources\n\n/opt/hostedtoolcache/Python/3.12.8/x64/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "driver",
    "time" : {
      "start" : 1738029408079,
      "stop" : 1738029408831,
      "duration" : 752
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "clear_allure_results_folder",
    "time" : {
      "start" : 1738028384997,
      "stop" : 1738028384997,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "main_page_open",
    "time" : {
      "start" : 1738029408831,
      "stop" : 1738029413444,
      "duration" : 4613
    },
    "status" : "passed",
    "steps" : [ {
      "name" : "Open page: https://brainup.site/",
      "time" : {
        "start" : 1738029408831,
        "stop" : 1738029413444,
        "duration" : 4613
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "driver",
        "value" : "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"92f1b8fbdabceb08c257bb1f8f32bdca\")>"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  } ],
  "testStage" : {
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
    "statusTrace" : "self = <pages.main_page.MainPage object at 0x7f8f5410de20>\nurl = 'https://brainup.site/used-resources'\n\n    def check_expected_link(self, url):\n        with allure.step(f'Check url is present: {url}'):\n            try:\n>               return Wait(self.driver, self.timeout).until(\n                    ec.url_to_be(url), message=f\"Can't find element by locator {url}\")\n\npages/base_page.py:68: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"92f1b8fbdabceb08c257bb1f8f32bdca\")>\nmethod = <function url_to_be.<locals>._predicate at 0x7f8f540ff240>\nmessage = \"Can't find element by locator https://brainup.site/used-resources\"\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources\n\n/opt/hostedtoolcache/Python/3.12.8/x64/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException\n\nDuring handling of the above exception, another exception occurred:\n\nself = <tests.main_page_test.TestMainPage object at 0x7f8f542f1310>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"92f1b8fbdabceb08c257bb1f8f32bdca\")>\nmain_page_open = None\n\n    @allure.title(\"Verify redirection to used resources page not authorized user.\")\n    def test_mp_08_verify_redirection_to_used_resources_page(self, driver, main_page_open):\n        page = MainPage(driver)\n>       resources_page_url = page.open_used_resources_page()\n\ntests/main_page_test.py:57: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/main_page.py:68: in open_used_resources_page\n    self.check_expected_link(MainPageLinks.URL_USED_RESOURCES_PAGE)\npages/base_page.py:72: in check_expected_link\n    return Wait(self.driver, self.timeout).until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"92f1b8fbdabceb08c257bb1f8f32bdca\")>\nmethod = <function url_to_be.<locals>._predicate at 0x7f8f540ff060>\nmessage = \"Can't find element by locator https://brainup.site/used-resources\"\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources\n\n/opt/hostedtoolcache/Python/3.12.8/x64/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException",
    "steps" : [ {
      "name" : "Open used resources page",
      "time" : {
        "start" : 1738029413444,
        "stop" : 1738029453868,
        "duration" : 40424
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.8/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/pages/main_page.py\", line 68, in open_used_resources_page\n    self.check_expected_link(MainPageLinks.URL_USED_RESOURCES_PAGE)\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/pages/base_page.py\", line 72, in check_expected_link\n    return Wait(self.driver, self.timeout).until(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.8/x64/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n",
      "steps" : [ {
        "name" : "Check url is present: https://brainup.site/used-resources",
        "time" : {
          "start" : 1738029413611,
          "stop" : 1738029453867,
          "duration" : 40256
        },
        "status" : "broken",
        "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources\n\n",
        "statusTrace" : "  File \"/home/runner/work/auto-tests-python/auto-tests-python/pages/base_page.py\", line 72, in check_expected_link\n    return Wait(self.driver, self.timeout).until(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.8/x64/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n",
        "steps" : [ ],
        "attachments" : [ ],
        "parameters" : [ ],
        "stepsCount" : 0,
        "attachmentsCount" : 0,
        "shouldDisplayMessage" : true,
        "hasContent" : true
      } ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 1,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : true
    } ],
    "attachments" : [ {
      "uid" : "4a3e120189de84e6",
      "name" : "Screenshot 2025-01-28 01:57:33.972758",
      "source" : "4a3e120189de84e6.png",
      "type" : "image/png",
      "size" : 58734
    } ],
    "parameters" : [ ],
    "stepsCount" : 2,
    "attachmentsCount" : 1,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  },
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1738029453974,
      "stop" : 1738029454043,
      "duration" : 69
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "Main Page."
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "main_page_test"
  }, {
    "name" : "subSuite",
    "value" : "TestMainPage"
  }, {
    "name" : "host",
    "value" : "fv-az1947-905"
  }, {
    "name" : "thread",
    "value" : "2489-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.main_page_test"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 36,
        "skipped" : 0,
        "passed" : 78,
        "unknown" : 0,
        "total" : 114
      },
      "items" : [ {
        "uid" : "9cc7859b90df7af7",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/167//#testresult/9cc7859b90df7af7",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1737943039169,
          "stop" : 1737943079624,
          "duration" : 40455
        }
      }, {
        "uid" : "bfe682ff3ff5f42d",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/166//#testresult/bfe682ff3ff5f42d",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1737856686196,
          "stop" : 1737856726724,
          "duration" : 40528
        }
      }, {
        "uid" : "c4eeaeae961dc349",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/165//#testresult/c4eeaeae961dc349",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1737769926835,
          "stop" : 1737769967270,
          "duration" : 40435
        }
      }, {
        "uid" : "9ae2a86931ee6185",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/164//#testresult/9ae2a86931ee6185",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1737683748802,
          "stop" : 1737683789305,
          "duration" : 40503
        }
      }, {
        "uid" : "3e349effac849c10",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/163//#testresult/3e349effac849c10",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1737597372232,
          "stop" : 1737597412675,
          "duration" : 40443
        }
      }, {
        "uid" : "28f9fb58b3dea366",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/162//#testresult/28f9fb58b3dea366",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1737511030651,
          "stop" : 1737511071098,
          "duration" : 40447
        }
      }, {
        "uid" : "ca41bc61628f3483",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/161//#testresult/ca41bc61628f3483",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1737424624527,
          "stop" : 1737424664972,
          "duration" : 40445
        }
      }, {
        "uid" : "945d9e5c7b5af6ce",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/160//#testresult/945d9e5c7b5af6ce",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1737338279543,
          "stop" : 1737338319980,
          "duration" : 40437
        }
      }, {
        "uid" : "f52c22771884382f",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/159//#testresult/f52c22771884382f",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1737252128835,
          "stop" : 1737252169308,
          "duration" : 40473
        }
      }, {
        "uid" : "3b367826c0ab7bda",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/158//#testresult/3b367826c0ab7bda",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1737165130824,
          "stop" : 1737165171299,
          "duration" : 40475
        }
      }, {
        "uid" : "afb69c35f85a9749",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/157//#testresult/afb69c35f85a9749",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1737078868818,
          "stop" : 1737078909285,
          "duration" : 40467
        }
      }, {
        "uid" : "79ea6a9a3d526a00",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/156//#testresult/79ea6a9a3d526a00",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1736992590352,
          "stop" : 1736992630805,
          "duration" : 40453
        }
      }, {
        "uid" : "a0fef04b4584f39b",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/155//#testresult/a0fef04b4584f39b",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1736906169963,
          "stop" : 1736906210407,
          "duration" : 40444
        }
      }, {
        "uid" : "9dd077cec2c79571",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/154//#testresult/9dd077cec2c79571",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1736819751846,
          "stop" : 1736819792331,
          "duration" : 40485
        }
      }, {
        "uid" : "2bb45850c7dfe9c1",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/153//#testresult/2bb45850c7dfe9c1",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1736733792035,
          "stop" : 1736733832510,
          "duration" : 40475
        }
      }, {
        "uid" : "c96e83ccfe73844f",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/152//#testresult/c96e83ccfe73844f",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1736647706235,
          "stop" : 1736647746691,
          "duration" : 40456
        }
      }, {
        "uid" : "7c235b44f87eba9b",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/151//#testresult/7c235b44f87eba9b",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1736560726255,
          "stop" : 1736560766688,
          "duration" : 40433
        }
      }, {
        "uid" : "982f85261286a508",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/150//#testresult/982f85261286a508",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1736474440159,
          "stop" : 1736474480624,
          "duration" : 40465
        }
      }, {
        "uid" : "cedc963273834e35",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/149//#testresult/cedc963273834e35",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1736389516174,
          "stop" : 1736389556636,
          "duration" : 40462
        }
      }, {
        "uid" : "cdc44bb285a6288",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/148//#testresult/cdc44bb285a6288",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1736301540196,
          "stop" : 1736301580668,
          "duration" : 40472
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "c7efec0d4fcde38b.json",
  "parameterValues" : [ ]
}